From 553d2243e2a42783b8bb94addfb752b802c23311 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Tue, 6 Jan 2015 00:29:40 -0500
Subject: tree-wide: remove unnecessary LOG_PRI

LOG_DEBUG is already a log level, there is no need to use LOG_PRI which
is for filtering out the facility.

diff --git a/src/core/execute.c b/src/core/execute.c
index 51e1e87..0af4227 100644
--- a/src/core/execute.c
+++ b/src/core/execute.c
@@ -797,7 +797,7 @@ static int setup_pam(
          * daemon. We do things this way to ensure that the main PID
          * of the daemon is the one we initially fork()ed. */
 
-        if (log_get_max_level() < LOG_PRI(LOG_DEBUG))
+        if (log_get_max_level() < LOG_DEBUG)
                 flags |= PAM_SILENT;
 
         pam_code = pam_start(name, user, &conv, &handle);
@@ -1782,7 +1782,7 @@ static int exec_child(ExecCommand *command,
 
         final_env = strv_env_clean(final_env);
 
-        if (_unlikely_(log_get_max_level() >= LOG_PRI(LOG_DEBUG))) {
+        if (_unlikely_(log_get_max_level() >= LOG_DEBUG)) {
                 _cleanup_free_ char *line;
 
                 line = exec_command_line(final_argv);
diff --git a/src/journal/coredumpctl.c b/src/journal/coredumpctl.c
index 756e793..afb39ad 100644
--- a/src/journal/coredumpctl.c
+++ b/src/journal/coredumpctl.c
@@ -822,7 +822,7 @@ int main(int argc, char *argv[]) {
                 }
         }
 
-        if (_unlikely_(log_get_max_level() >= LOG_PRI(LOG_DEBUG))) {
+        if (_unlikely_(log_get_max_level() >= LOG_DEBUG)) {
                 _cleanup_free_ char *filter;
 
                 filter = journal_make_match_string(j);
diff --git a/src/journal/journalctl.c b/src/journal/journalctl.c
index c91f2cf..386e350 100644
--- a/src/journal/journalctl.c
+++ b/src/journal/journalctl.c
@@ -1875,7 +1875,7 @@ int main(int argc, char *argv[]) {
                 return EXIT_FAILURE;
         }
 
-        if (_unlikely_(log_get_max_level() >= LOG_PRI(LOG_DEBUG))) {
+        if (_unlikely_(log_get_max_level() >= LOG_DEBUG)) {
                 _cleanup_free_ char *filter;
 
                 filter = journal_make_match_string(j);
diff --git a/src/shared/logs-show.c b/src/shared/logs-show.c
index 98972ed..dff0d30 100644
--- a/src/shared/logs-show.c
+++ b/src/shared/logs-show.c
@@ -1263,7 +1263,7 @@ int show_journal_by_unit(
         if (r < 0)
                 return r;
 
-        if (_unlikely_(log_get_max_level() >= LOG_PRI(LOG_DEBUG))) {
+        if (_unlikely_(log_get_max_level() >= LOG_DEBUG)) {
                 _cleanup_free_ char *filter;
 
                 filter = journal_make_match_string(j);
-- 
cgit v0.10.2

